pkgs/by-name/ki/kimai/package.nix

```diff
@@ -0,0 +1,61 @@
+{ php, fetchFromGitHub, lib, nixosTests }:
+
+php.buildComposerProject (finalAttrs: {
+  pname = "kimai";
+  version = "2.24.0";
+
+  src = fetchFromGitHub {
+    owner = "kimai";
+    repo = "kimai";
+    rev = finalAttrs.version;
+    hash = "sha256-C6i263sAfZwZELUIcZh/OmXZqgCKifjPYBafnH0wMC4=";
+  };
+
+  php = php.buildEnv {
+    extensions = ({ enabled, all }: enabled ++ (with all; [
+      gd
+      intl
+      mbstring
+      pdo
+      tokenizer
+      xml
+      xsl
+      zip
+    ]));
+
+    # Asset building and (later) cache building process requires a little bit
+    # more memory.
+    extraConfig = ''
+      memory_limit=384M
+    '';
+  };
+
+  vendorHash = "sha256-3y3FfSUuDyBGP1dsuzDORDqFNj3jYix5ArM+2FS4gn4=";
+
+  composerNoPlugins = false;
+  composerNoScripts = false;
+
+  postInstall = ''
+    # Make available the console utility, as Kimai doesn't list this in
+    # composer.json.
+    mkdir -p "$out"/share/php/kimai "$out"/bin
+    ln -s "$out"/share/php/kimai/bin/console "$out"/bin/console
+  '';
+
+  passthru.tests = {
+    kimai = nixosTests.kimai;
+  }
```

Comment:
Fails to parse here

```
error: syntax error, unexpected '=', expecting ';'
       at /home/runner/work/nixpkgs/nixpkgs/pkgs/by-name/ki/kimai/package.nix:49:8:
           48|
           49|   meta = {
             |        ^
           50|     description = "A web-based multi-user time-tracking application";
```

```suggestion
  };
```

Reply:
Well... the error was introduced when I was trying to add `passthru.tests` attribute last-minute. Should have tested the final version before submitting. Fixed.